### PR TITLE
task/configuration/static_config

### PR DESCRIPTION
**Pull request by BarAgam**
Refactored code to use the static configuration file - Constants.java

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/teamagam/GimelGimel/pull/50" target="_blank">ReviewNinja

File: app/src/debug/java/com/teamagam/gimelgimel/app/utils/Constants.java [hunk 1]
```diff
@@ -1,13 +1,24 @@
...
 
 public class Constants {
-
-    // Just examples. Replace however you need!
-    public static final int HTTP_CONNECT_TIMEOUT =  6000; // milliseconds
-    public static final int HTTP_READ_TIMEOUT = 10000; // milliseconds
+    public static final String MESSAGING_SERVER_URL = "http://ggmessaging-dev1.herokuapp.com";
 
     public static final String build = "debug";
     public static final int a = 1;
```
Comment (teamagam1) [inline app/src/debug/java/com/teamagam/gimelgimel/app/utils/Constants.java]: what is "a" for? !fix

File: app/src/debug/java/com/teamagam/gimelgimel/app/utils/Constants.java [hunk 2]
```diff
@@ -1,13 +1,24 @@
 package com.teamagam.gimelgimel.app.utils;
 
 public class Constants {
-
-    // Just examples. Replace however you need!
-    public static final int HTTP_CONNECT_TIMEOUT =  6000; // milliseconds
-    public static final int HTTP_READ_TIMEOUT = 10000; // milliseconds
+    public static final String MESSAGING_SERVER_URL = "http://ggmessaging-dev1.herokuapp.com";
 
     public static final String build = "debug";
```
Comment (teamagam1) [inline app/src/debug/java/com/teamagam/gimelgimel/app/utils/Constants.java]: you have deleted this constant in the release file. choose one - either delete both or keep both. !resolve

Comment (BarAgam) [inline app/src/debug/java/com/teamagam/gimelgimel/app/utils/Constants.java]: !resolved

File: app/src/debug/java/com/teamagam/gimelgimel/app/utils/Constants.java (hunk 1 above)
Comment (BarAgam) [inline app/src/debug/java/com/teamagam/gimelgimel/app/utils/Constants.java]: !resolved

**Pull request merged**